src/components/Dashboard/styled.tsx

```diff
@@ -0,0 +1,32 @@
+import { lg, black500, extraLargeFontSize } from 'src/theme/variables'
+import styled from 'styled-components'
+
+export const WidgetContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`
+
+export const WidgetTitle = styled.h2`
+  color: ${black500};
+  font-size: ${extraLargeFontSize};
+  margin-top: 0;
+`
+
+export const WidgetBody = styled.div<{ filled?: boolean }>`
```

Comment:
You're not using `filled` here.

Reply:
Removed 👍  it was a leftover from a previous style